tests/unit/validationMiddlewareTest.js

```diff
@@ -72,13 +69,11 @@ tape.test('Validation Middleware', t => {
         }
       }
 
-      performValidation(ctx, joiSchema)
-
-      t.notEqual(ctx.body.toString().match(/"name" is required/).length, 0)
+      t.throws(() => performValidation(ctx, joiSchema), `Validation execution failed: "name" is required`)
       t.end()
     })
 
-    t.test('should set the ctx input property when validation succeeds', t => {
+    t.test('should set not throw when validation succeeds', t => {
```

Comment:
they way this test description reads seems a bit odd. I think we can remove the `... set ...` part